gen/py/chromestatus_openapi/chromestatus_openapi/openapi/openapi.yaml

```diff
@@ -57,6 +57,7 @@ paths:
           description: Account not found
       summary: Delete an account
       x-openapi-router-controller: chromestatus_openapi.controllers.default_controller
+  /activities: {}
```

Comment:
I was thinking that this would have a specification of the start parameter and the fact that the 200 response will be a GetReviewActivitiesResponse.

Reply:
That's true, I missed part of the definition. It's been added!